source/includes/_subscriptions.md

```diff
@@ -439,3 +439,101 @@ size | `number` | Number of subscriptions per page _default is 10_
 page | `number` | Which page to retrieve _default is 0_
 sort | `string` | Field used to sort results _default is `-modified`_
 status | `string` | The subscription status _default is `ACTIVE`_
+
+
+## Recurring Order Subscriptions Object
+
+Create this type of plan when you need to schedule some [`Orders`](#Orders) daily, weekly, or monthly.
+To plan a schedule follow the next instruction on [`Recurring Order Plan](#recurring-order-plan).
+
+Attribute | Type | Description
+--------- | ---- | -------
+_id | `object` | The subscription ID.
+method | `string` | Name of subscription method. `recurring_order`
+name | `string` | The name of the subscription type. _default is `RECURRING_ORDER`_
+note | `string` | A short description of the description.
+user | `object`  | [`User`](#Users) ID associated with the subscription.
+status | `string` | Status for the subscription. _default is `CREATED`, also available are `ACTIVE`, `FUTURE`, `NON_RENEWING`and `CANCELLED`_
```

Comment:
Status of the subscription. This will change according to the subscription's state. _default is `CREATED`._